quinn-proto/src/transport_parameters.rs

```diff
@@ -99,6 +99,9 @@ macro_rules! make_struct {
             pub(crate) stateless_reset_token: Option<ResetToken>,
             /// The server's preferred address for communication after handshake completion
             pub(crate) preferred_address: Option<PreferredAddress>,
+
+
+            pub(crate) ack_timestamps_cfg: AckTimestampsConfig,
```

Comment:
New transport parameters must document whether they are preserved across 0-RTT or not. What is the case here? Note that e.g. `ack_delay_exponent` is *not* preserved.